README.md

```diff
@@ -12,6 +12,19 @@ Yotas are set by one of OSS Cameroon admin on this [spreadsheet](https://docs.go
 ## How to run locally
 Run `yarn dev`
```

Comment:
Can you add the `yarn install` step in the documentation, as you mentioned that it was not present in here ?